src/styles/variables/variables.scss

```
@@ -0,0 +1,13 @@
+$bg-color: #fff;
+$main-font-color: #060b35;
+$secondary-font-color: #616070;
+$btn-gb-color: #00acdc;
+$btn-color: #00acdc;
+$border-color: #f3f3f3;
+$links-color: #060b35;
+$bg-color: rgba(255, 255, 255, 1);
+$hover-bg: #00acdc;
+$hover-title-color: #34568b;
+$hover-link-color: #00acdc;
+$hover-bg-btn-color: #fff;
+$hover-btn-color: #00acdc;
```

Remove all duplicates
```suggestion
$bg-color: #fff;
$main-font-color: #060b35;
$secondary-font-color: #616070;
$btn-gb-color: #00acdc;
$border-color: #f3f3f3;
$bg-color: rgba(255, 255, 255, 1);
$hover-title-color: #34568b;
```